openid-caep-interoperability-profile-1_0.md

```diff
@@ -0,0 +1,193 @@
+    title: OpenID Continuous Access Evaluation Profile 1.0
+    author:
+      -
+        ins: T. Cappalli
+        name: Tim Cappalli
+        org: Microsoft
+      -
+        ins: A. Tulshibagwale
+        name: Atul Tulshibagwale
+        org: SGNL
+  RFC8935:
+
+
+--- abstract
+This document defines an interoperability profile for implementations of the Shared Signals Framework (SSF) {{SSF}} and the Continuous Access Evaluation Profile (CAEP) {{CAEP}}. It is organized around use-cases that improve security of authenticated sessions. It specifies certain optional elements from within the SSF and CAEP specifications as being required to be supported in order to be considered as an interoperable implementation.
+
+--- middle
+
+# Introduction {#introduction}
+SSF and CAEP together enable improved session security outcomes. This specification defines the minimum required features from SSF and CAEP that an implementation MUST offer in order to be considered as an interoperable implementation. This document defines specific use cases. An implementation may support only a subset of the use cases defined herein, and SHALL be considered an interoperable implementation for the specific use-cases it supports. The following use-cases are considered as a part of this specification:
+
+Session Revocation
+: A SSF Transmitter or Receiver is able to respectively generate or respond to the CAEP session-revoked event
+
+Credential Change
+: A SSF Transmitter or Receiver is able to respectively generate or respond to the CAEP credential-change event
+
+# Common Requirements {#common-requirements}
+The following requirements are common across all use-cases defined in this document.
+
+## Transmitters {#common-transmitters}
+Transmitters MUST implement the following features:
+
+### Spec Version {#spec-version}
+The Transmitter Configuration Metadata MUST have a `spec_version` field, and its value MUST be `1_0-ID2` or greater
+
+### Delivery Method {#delivery-method}
+The Transmitter Configuration Metadata MUST include the `delivery_methods_supported` field.
+
+### JWKS URI {#jwks-uri}
+The Transmitter Configuration Metadata MUST include the `jwks_uri` field, and its value MUST provide the current signing key of the Transmitter.
+
+### Configuration Endpoint {#configuration-endpoint}
+The Transmitter Configuration Metadata MUST include the `configuration_endpoint` field. The specified endpoint MUST provide a way to Create a Stream.
+
+### Status Endpoint {#status-endpoint}
+The Transmitter Configuration Metadata MUST include the `status_endpoint` field. The specified endpoint MUST provide a way to Get and Update the Stream Status. The Transmitter MUST be able to pause and restart streams. For streams that are paused, the Transmitter MUST specify (offline) the resource constraints on how many events it can keep, or for how long. The way a Transmitter specifies this information is outside the scope of the SSF spec.
```

Comment:
```suggestion
The Transmitter Configuration Metadata MUST include the `status_endpoint` field. The specified endpoint MUST provide a way to get and update the stream status. The Transmitter MUST be able to pause and restart streams. For streams that are paused, the Transmitter MUST specify (offline) the resource constraints on how many events it can keep, or for how long. The way a Transmitter specifies this information is outside the scope of the SSF spec.
```